.github/workflows/build.yaml

```
@@ -372,7 +372,8 @@ jobs:
         if: |
           github.event_name == 'push' || !github.event.pull_request.head.repo.fork
         run: |
-          scripts/lib.sh retry 5 true docker login -u "${QUAY_RHACS_ENG_RO_USERNAME}" --password-stdin quay.io <<<"${QUAY_RHACS_ENG_RO_PASSWORD}"
+          source ./scripts/ci/lib.sh
+          registry_ro_login "quay.io/rhacs-eng"
```

This could even be:
```suggestion
          ./scripts/ci/lib.sh registry_ro_login "quay.io/rhacs-eng"
```
Right?
Similar below.

---

In other place we are using docker/login-action maybe we can use it here instead
https://github.com/stackrox/stackrox/blob/707e046b929a9ab49c2ff29dffb5cb5bb8021944/.github/workflows/verify-release.yml#L73
alternatively does it make sense to create our own action that could be reused here and there?

---

yeah, lets do it.